src/odinapi/database/level2db.py

```diff
@@ -376,7 +380,7 @@ def get_measurements(self, products,
         sort = [('FreqMode', ASCENDING), ('ScanID', ASCENDING)]
 
         for conc in self.L2_collection.find(
-                query, fields, sort=sort, limit=HARD_LIMIT):
+                query, fields, sort=sort, limit=document_limit):
```

Comment:
I don't know what the reason for the value of the HARD_LIMIT is, if it relates to the database query, the limiting should really be handled here. If it relates to the size of the data transfer to whomever requests the data, then it should be handled in the endpoint. As it was written here, I assumed it was a database issue.

Now limit can be set to `None`, is that really sound? looks like it ought to be `HARD_LIMIT if document_limit is None else min(HARD_LIMIT, document_limit)` and that value still needs to be returned from here as long as document_limit is a keyword argument/has default value. If `limit=None` is fine, then add tests for that.

If you don't want that, the function needs to be:
```
def get_measurements(self, products, document_limit, ....)
```
And then `HARD_LIMIT` should probably be all together be removed from this module.

Reply:
The hard limit is still used, but not by the get_measurements function here and the endpoints we were supposed to fix in this task. 

 changed get_measurements input according to your suggestion

Reply:
why can't they all use the same limit, is there a reason why they should be different limits?

Reply:
no. now they use the same limit.